specification/iotcentral/resource-manager/readme.go.md

```diff
@@ -0,0 +1,36 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  clear-output-folder: true
+  namespace: iotcentral
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2018-09-01
+  - tag: package-2017-07-01-privatepreview
+```
+
+### Tag: package-2018-09-01 and go
+
+These settings apply only when `--tag=package-2018-09-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2018-09-01' && $(go)
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2018-09-01/$(namespace)
+```
+
+### Tag: package-2017-07-01-privatepreview and go
```

Comment:
really ? should we even leave that in ?

Reply:
im talking about hte privatepreview api

Reply:
@emgarten do we need to keep building this API version?